lib/domain/document-request.js

```diff
@@ -0,0 +1,21 @@
+const { nanoid } = require('nanoid');
+
+class DocumentRequest {
+  constructor({ id, created, dropboxId, metadata }) {
+    this.id = id;
+    this.created = created;
+    this.dropboxId = dropboxId;
+    this.metadata = metadata;
+  }
+
+  static new(metadata) {
```

Comment:
Personal preference, but I usually go with `.create` for static methods that create new instances, to avoid any confusion between `new DocumentRequest(...)` and make it obvious it isn't just doing what a constructor would do.